src/ui/component/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,49 @@
+import type { FC } from 'react'
+import { useCallback } from 'react'
+import classNames from 'classnames'
+import * as RCheckbox from '@radix-ui/react-checkbox'
+import { Icon } from '@/ui/component/icon/icon'
+import './checkbox.scss'
+
+export type CheckboxOption = {
+  id: string
+  name: string
+  checked: boolean
+  disabled: boolean
+}
+
+type CheckboxProps = {
+  id: string
+  name: string
+  checked: boolean
+  disabled: boolean
```

Comment:
No sure that we need to have disabled mandatory, could you have it optional with a default to false?

Reply:
Sure, thanks! I realised that I didn't set the `disable` prop to the checkbox. I will fix that.